test/unit/events_test.js

```diff
@@ -12,9 +12,14 @@ describe('humanizing raw events', function () {
   const data1 = nativeToScVal({ hello: 'world' });
 
   // workaround for xdr.ContractEventBody.0(...) being invalid lol
-  const bodyModel = xdr.ContractEventBody.fromXDR('AAAAAAAAAAAAAAAB', 'base64');
   const cloneAndSet = (newBody) => {
-    const clone = xdr.ContractEventBody.fromXDR(bodyModel.toXDR());
+    const clone = new xdr.ContractEventBody(
+      0,
+      new xdr.ContractEventV0({
+        topics: [],
+        data: xdr.ScVal.scvVoid()
+      })
+    );
```

Comment:
it doesn't look like test is exercising omitted contractId behavior, can another test be added to assert if this is important?

Reply:
ooh great point, thanks :pray: